src/Physics.hs

```diff
@@ -56,7 +59,7 @@ module Physics
 
   -- * Brick Universe
 
-  -- | Detect a collision with one a the bricks still present. Upon collisions,
+  -- | Detect a collision with one of the bricks list. Upon collisions,
```

Comment:
I don't like that description, nor the name of this function actually. It should be renamed: `handleBallBrickCollision`, for example, as it implements all the business logic rules as a `Game` transformation:

- Make the brick disappear.
- Make ball bounce.
- Increase ball speed.